### DISCUSSION TITLE
Pra quem for do Paraná: pesquisa sobre salários no estado

### DISCUSSION THREAD

**Discussion by juliovedovatto** [frontendbr/forum#1129] [Discussão]
Pessoal do Paraná, pode ajudar?

https://goo.gl/forms/3o7EiRMlLKe1uXm93

```
A comunidade do DevParaná (devparana.org) está realizando uma pesquisa salarial, com intuito acadêmico, para ver a realidade do desenvolvedor aqui no estado.

https://goo.gl/forms/3o7EiRMlLKe1uXm93

Queremos pedir sua participação nessa pesquisa! 🤓

Ela tem poucas perguntas e poucos campos são obrigatórios. Você pode omitir o nome da empresa que trabalha, por exemplo.

Os resultados serão usados como métrica para uma discussão no próximo BEMUG (encontro Backend de Maringá)

Link do Meetup: https://www.meetup.com/developerparana/events/252166359/
```

**Comment by vsppedro** [comment]
Mas depois esses dados sobre os salários irão ficar abertos?

É que vejo como uma fonte de informação interessante para quem encontrar oportunidades de trabalho no Paraná, pois ao ver o salário e a empresa o candidato já vai ter em mente o que esperar. E quem sabe até ajudar a decidir a resposta para a velha pergunta em entrevistas: "Qual a sua pretensão salarial?".

**Comment by lidimayra** [comment]
No Love Mondays, é possível filtrar cargos por cidade... aqui, por exemplo:
https://www.lovemondays.com.br/salarios/cargo/pr/salario-desenvolvedor/maringa

É possível conferir os salários reportados por desenvolvedores em Maringá, PR :wink:

**Comment by juliovedovatto** [comment]
@VSPPedro, sim vai ficar disponível os dados. Transparência acima de tudo.

@lidimayra O lovemondays não uma fonte boa pra se ter base salarial. Principalmente que existe um acordo entre empresas aqui pra manter uma média salarial baixa. Só empresas que vem de fora aqui em Maringá oferecem salário acima do mercado.

**Comment by lidimayra** [comment]
Olá @juliovedovatto, tudo bom?
Os dados do Love Mondays são gerados pelos usuários da plataforma, não pelas empresas. Qualquer usuário é livre pra [enviar o seu próprio salário](https://www.lovemondays.com.br/contribua/salario), mantendo a base cada vez maior e mais próxima da realidade. :blush:

**Comment by juliovedovatto** [comment]
sim @lidimayra, sou ciente deste fato.

Em conversa com diversos profissionais que conheço, chegamos a conclusão que o Love Mondays não se pode ser tomado como base de análise salarial. Um dos fatores é que nem sempre quem preenche o salário tem ainda aquele salário, ou já saiu do trabalho ou está a X anos trabalhando pra empresa.

Tenho conhecimento de como funciona o processo e transparência do serviço. Os criadores do formulário levaram isso em consideração e decidiram realizar um senso salarial do 0, para ser usado em discussões do grupo.

**Comment by lidimayra** [comment]
Sua opinião @juliovedovatto, então respeito embora não concorde.

O Love Mondays possui uma equipe especializada em moderação de conteúdo dedicada em realizar a verificação dos dados. A plataforma aplica ajustes de inflação, expira salários antigos, verifica valores extremos, confere casos suspeitos... enfim, é um sistema bastante complexo para atender as necessidades de forma bastante democrática e cautelosa.

Há cuidados que são tomados que o formulário, por exemplo, não é capaz. Pelo contrário, fica muito mais fácil de registrar uma informação falsa neste caso.

No mais, desejo sorte na sua pesquisa e sucesso!

**Comment by juliovedovatto** [comment]
Resultados: https://docs.google.com/spreadsheets/d/16goUGes8FSc_R1culImVk__dq1s9r2fdy_0lEXIgJek/edit#gid=481182473
